### ISSUE TITLE
Suggestion: Linking specific combinations of items through URL

### ISSUE THREAD

**Issue by mclifford82** [kevindqc/mrrl#3]
I think it would be really cool to be able to link a specific days combination of items to my guildies without them having to put in the numbers themselves.

My thought is that it could be achieved through a url formed as such: https://mrrl.io/?00004000103

Each digit representing one item in the available rotation, the value representing the quantity. This would then auto-fill 4 Particularly Dense Rocks, 1 Extra-Slimy Snail and 3 Sea Giant Food Dust on the web form.

**Comment by jacobjdowning** [comment]
I'd like to implement this enhancement, are you open to pull requests @kevindqc ?

**Comment by kevindqc** [comment]
Yeah! Sorry the code is not the best, it was my first time using React
